api/profile/src/config.rs

```diff
@@ -0,0 +1,16 @@
+use spin_sdk::config::get;
+
+const KEY_DB_URL: &str = "db_url";
+
+#[derive(Debug)]
+pub(crate) struct Config {
+    pub db_url: String,
+}
+
+impl Config {
+    pub(crate) fn get() -> Config {
+        Config {
+            db_url: get(KEY_DB_URL).unwrap(),
```

Comment:
I think i buy the infallibility of `get(KEY_DB_URL).unwrap()` here but because i am a little skeptical i think i'd use `expect`, e.g. `.expect("database url")`.

Reply:
What's the difference? Is it just customizing the error message with unwrap?

Reply:
It just adds context to the panic which is a little easier to grok than the stack trace (if there is one)